lib/supabase/getChannelStats.ts

```diff
@@ -0,0 +1,69 @@
+import { CHANNELS } from "../consts";
+import { supabaseClient } from "./client";
+
+type ChannelAccumulator = Record<string, {
+  uniqueAuthors: Set<number>;
+  uniquePosts: Set<string>;
+  notes?: number;
+}>;
+
+interface Stats {
+  channelId: string;
+  numberOfCurators: number;
+  numberOfSongs: number;
+  numberOfNotes?: number
+};
```

Comment:
can you please move `type` definitions into either a `standalone` or `shared` `types` file?